components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Storage/PublishedNodesProvider.cs

```diff
@@ -0,0 +1,159 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Models;
+using Serilog;
+using System;
+using System.IO;
+using System.Text;
+using System.Threading;
```

Comment:
using statements should be inside of the namespace block. like everywhere else

Reply:
Fixed.